epochStart/metachain/systemSCs.go

```diff
@@ -714,6 +859,7 @@ func (s *systemSCProcessor) updateMaxNodes(validatorInfos map[uint32][]*state.Va
 		return epochStart.ErrInvalidMaxNumberOfNodes
 	}
 
+	// TODO: Check if flag is not enabled, should we move staked nodes to AuctionList?
```

Comment:
TODO still relevant?

Reply:
Hmmm, actually it is not relevant. Removed it